src/app/components/selectable-box/SelectableBoxController.test.js

```diff
@@ -1,7 +1,8 @@
 import React from "react";
-import SelectableBoxController from "./SelectableBoxController.jsx";
+import SelectableBoxController from "./SelectableBoxController";
 import renderer from "react-test-renderer";
 import { shallow, mount } from "enzyme";
+import JestSnapshot from "../../../node_modules/jest-snapshot/build/index";
```

Comment:
More of a question than anything, Does this need to be imported here for anything?
I can see it was using snapshots before without this import.

Reply:
oh that is bad.. it was added by auto typing by my VSC, bad boy ... I missed it